src/styles/app.styl

```diff
@@ -42,8 +42,13 @@ body
 
 .KonnectorErrors
     max-width 662px // same as https://github.com/cozy/cozy-home/commit/10d4f4e605a2b5308dac9921538f194db77b7540#diff-17941d0d4818675dfedc84955549bdf2R60
-    width 100%
-    margin 3rem auto 0
+    margin .5rem auto 0
+
+    p
+        margin .5rem 0
+
+    +tiny-screen()
+        margin 1rem 1rem 0
```

Comment:
Should it be a spacing unit instead?

Reply:
I was also wondering if the shouldn't use cozy-ui's spacing_values in apps styles. Maybe a small topic for the front meeting this morning?